DailyScreenshot/ModEntry.cs

```diff
@@ -384,6 +384,54 @@ private void ReportLoadingError()
         /// <param name="e">The event data.</param>
         private void OnGameLaunched(object sender, GameLaunchedEventArgs e)
         {
+            // add Generic Mod Config Menu integration
+            var gmcmApi = Helper.ModRegistry.GetApi<GenericModConfigMenuAPI>("spacechase0.GenericModConfigMenu");
+            if (gmcmApi != null)
+            {
+                gmcmApi.RegisterModConfig(ModManifest, () => m_config = new ModConfig(), () => Helper.WriteConfig(m_config));
+                gmcmApi.RegisterLabel(ModManifest, "Effect control", "Toggel auditory and visual effects as well as notifications.");
+
+                gmcmApi.RegisterSimpleOption(
+                    ModManifest,
+                    "Auditory effects",
+                    "Toggles if a camera sound plays whenever a screenshot was taken.",
+                    () => m_config.auditoryEffects,
+                    (bool val) => m_config.auditoryEffects = val
+                );
+
+                gmcmApi.RegisterSimpleOption(
+                    ModManifest,
+                    "Visual effects",
+                    "Toggles if the screen flashes whenever a screenshot was taken.",
+                    () => m_config.visualEffects,
+                    (bool val) => m_config.visualEffects = val
+                );
+
+                gmcmApi.RegisterSimpleOption(
+                    ModManifest,
+                    "Notifications",
+                    "Toggles if a notification is displayed whenever a screenshot was taken.",
+                    () => m_config.screenshotNotifications,
+                    (bool val) => m_config.screenshotNotifications = val
+                );
+
+
+                gmcmApi.RegisterLabel(
+                    ModManifest, 
+                    "!!!DISCLAIMER!!! (Hover to reveal)",
```

Comment:
Nice job drawing attention to this. Hopefully we'll be able to add the remaining config options before releasing the next update, in which case we should be able to remove this disclaimer. Otherwise we'll definitely want to include it.